integration-tests/api/__tests__/admin/order.js

```diff
@@ -1555,6 +1555,97 @@ describe("/admin/orders", () => {
       expect(response.status).toEqual(200)
     })
 
+    describe("Given an existing discount order", () => {
+      describe("When a store operator attemps to create a swap form the discount order", () => {
+        it("Then should successfully create the swap", async () => {
+          const api = useApi()
+
+          const response = await api.post(
+            "/admin/orders/test-order/swaps",
+            {
+              return_items: [
+                {
+                  item_id: "test-item",
+                  quantity: 1,
+                },
+              ],
+              additional_items: [{ variant_id: "test-variant-2", quantity: 1 }],
+            },
+            {
+              headers: {
+                authorization: "Bearer test_token",
+              },
+            }
+          )
+
+          const swapCartId = response.data.order.swaps[0].cart_id
+
+          const swapCartRes = await api.get(`/store/carts/${swapCartId}`, {
+            headers: {
+              authorization: "Bearer test_token",
+            },
+          })
+          const cart = swapCartRes.data.cart
+
+          expect(response.status).toEqual(200)
+          expect(cart.items.length).toEqual(2)
+          expect(cart.items).toEqual(
+            expect.arrayContaining([
+              expect.objectContaining({
+                unit_price: -8000,
+                adjustments: [
+                  expect.objectContaining({
+                    amount: -800,
+                  }),
+                ],
+              }),
+              expect.objectContaining({
+                unit_price: 8000,
+                adjustments: [
+                  expect.objectContaining({
+                    amount: 800,
+                  }),
+                ],
+              }),
+            ])
+          )
+          expect(cart.total).toEqual(0)
+        })
+      })
+
+      describe("And given a swap cart", () => {
+        describe("When a line item is added to the swap cart", () => {
+          it("Then should not delete existing return line item adjustments", async () => {
+            const api = useApi()
+
+            const response = await api.post(
+              `/store/carts/disc-swap-cart/line-items`,
```

Comment:
nit (if-minor): use `simpleSwapFactory` or "build" the swap with the API e.g. make the same calls as on lines 1563-1588 above, and then continue with the same cart id. 

I think we would benefit from generally moving away from the "seeder" files and to something more self-contained like the factories/API sequences.